DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -208,6 +208,40 @@ void UDolbyIOSubsystem::Initialize(const FString& Token)
 			            BroadcastEvent(OnVideoTrackRemoved, VideoTrack);
 		            });
 	        })
+	    .then(
+	        [this](event_handler_id)
+	        {
+		        return Sdk->device_management().add_event_handler(
+		            [this](const audio_device_changed& Event)
+		            {
+			            auto ChangedDirection = ToEDolbyIODeviceDirection(Event.utilized_direction);
```

Comment:
`const audio_device::direction`

Reply:
here `EDolbyIODeviceDirection` is needed or you don't like auto?

Reply:
refactored out